experimental/dds/tree2/src/test/shared-tree-core/editManager.spec.ts

```diff
@@ -269,24 +269,24 @@ describe("EditManager", () => {
 
 			it("Evicts trunk commits according to a provided minimum sequence number", () => {
 				const { manager } = editManagerFactory({});
-				for (let i = 0; i < 10; ++i) {
+				for (let i = 1; i <= 10; ++i) {
 					manager.addSequencedChange(applyLocalCommit(manager), brand(i), brand(i - 1));
 				}
 
 				assert.equal(manager.getTrunkChanges().length, 10);
 				manager.advanceMinimumSequenceNumber(brand(5));
-				assert.equal(manager.getTrunkChanges().length, 5);
+				assert.equal(manager.getTrunkChanges().length, 6);
 				manager.advanceMinimumSequenceNumber(brand(10));
-				assert.equal(manager.getTrunkChanges().length, 0);
-				for (let i = 10; i < 20; ++i) {
+				assert.equal(manager.getTrunkChanges().length, 1);
+				for (let i = 11; i <= 20; ++i) {
 					manager.addSequencedChange(applyLocalCommit(manager), brand(i), brand(i - 1));
```

Comment:
This test was slightly wrong before. Note how it used to be handing the EditManager a commit with sequence number 10, but the minimum sequence number was already 10. That shouldn't be possible; if the local client knows about minimum sequence number 10, then it should already have sequenced commit 10.